sensor_msgs/Image[] inhand_roi_arr